lib/rex/proto/dns/server.rb

```diff
@@ -312,7 +183,7 @@ def default_dispatch_request(cli,data)
       req.header.rCode = Dnsruby::RCode::NOERROR
     end
     req.header.qr = true # Set response bit
-    send_response(cli, validate_packet(req).data)
```

Comment:
Any reason this and similar `validate_packet` calls were dropped? The only reason I can see is that this function might not be defined anymore, but correct me if I'm missing something here.

Reply:
The packet validation stuff doesn't play now that we're on DNSRuby. Eventually i'd love to get everything standardized on just that one library